site/search/SearchChartHitRichDataLargeVariantHelpers.test.ts

```diff
@@ -0,0 +1,137 @@
+import { describe, it, expect } from "vitest"
+import { GRAPHER_TAB_NAMES } from "@ourworldindata/types"
+import { LargeVariantGridSlot } from "./SearchChartHitRichDataTypes.js"
+import { placeGrapherTabsInLargeVariantGrid } from "./SearchChartHitRichDataLargeVariantHelpers.js"
+
+describe(placeGrapherTabsInLargeVariantGrid, () => {
```

Comment:
I think [`describe`](https://vitest.dev/api/#describe) needs a string as the argument, doesn't it?

Reply:
We use this throughout the code base. It works (it just serialises to the name) and it keeps the name in sync